data/mods/MindOverMatter/effectoncondition/eoc_nether_attunement_events.json

```diff
@@ -522,6 +524,36 @@
       }
     ]
   },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_DRAIN_EFFECT_CHECK_HALLUCINATIONS",
+    "condition": {
+      "and": [
+        { "math": [ "u_vitamin('vitamin_psionic_drain')", ">=", "125" ] },
+        {
+          "x_in_y_chance": {
+            "x": {
+              "math": [
+                "clamp( (u_vitamin('vitamin_psionic_drain') - 280), 0, 50) + (nether_attune_difficulty_scaler(u_latest_channeled_power_difficulty)) + 10"
```

Comment:
The vitamin only goes up to 250, so this would mean there's only ever a 1% chance of hallucinations